README.md

```diff
@@ -1,44 +1,60 @@
-* automatic pagination
+* Generated functions provide full type checking and support for all options provided by the underlying GraphQL API.
+* Pagination is handled automatically, so you can easily retrieve more than 1,000 records per call.
+* Create standard await/async function or [react-query](https://react-query.tanstack.com/) functions using the `--method reactquery` option.
 
 ## Usage
 
-1. Enter your project's directory, and run `npm i --save codegen-graph-ts` (note that retention of this dependency is required for the generated code to function)
-2. Download the subgraph's manifest. To do so, find the subgraph's API url (located under "Queries (HTTP)" on subgraph page), and then run: `npx codegen-graph-ts pull <url> > manifest.json`
-3. Generate the code from your downloaded manifest: `npx codegen-graph-ts gen -s manifest.json -o subgraph.ts`
-4. You now have a typescript file, `subgraph.ts` which can be used to query data. For example, the below code snippet can be used to download top 1500 user balances in a hypothetical subgraph, and print them out:
+Replace the URL below with the *Queries (HTTP)* endpoint from The Graph. Then run the code snippet to create a `subgraph.ts` file. Note that you’ll need to keep the npm dependency in your project for the generated file to function. To generate JavaScript functions, rather than Typescript, add the `--js` flag to the last command. 
+
+```
+npm i --save codegen-graph-ts
+npx codegen-graph-ts pull https://api.thegraph.com/subgraphs/name/example-team/example-subgraph > manifest.json
+npx codegen-graph-ts gen -s manifest.json -o subgraph.ts
+```
+
+Now you now have a TypeScript file, `subgraph.ts`, which can be used to query data. Review the file for the names of the generated functions.
+
+The code snippet below could be used to display the top 1,500 user balances from a hypothetical subgraph at the example endpoint:
 
 ```
 import { getManyUserBalance } from './subgraph';
 
-const MY_SUBGRAPH_URL = 'https://thegraph...';
+const 
+SUBGRAPH_URL = 'https://api.thegraph.com/subgraphs/name/example-team/example-subgraph';
 
 const func = async () => {
-    const topBalances = await getManyUserBalance(MY_SUBGRAPH_URL, { first: 1500, orderBy: 'balanceOf', orderDirection: 'desc' }, { id: true, balanceOf: true, token: { symbol: true, name: true } });
-
-    console.log('top balances:');
+    const topBalances = await getManyUserBalance(
+        SUBGRAPH_URL,
+        {
+            first: 1500,
+            orderBy: 'balanceOf',
+            orderDirection: 'desc'
+        }, {
+            id: true,
+            balanceOf: true,
+            token: { symbol: true, name: true }
+        }
+    );
+
+    console.log('Top Balances');
     for (const entry of topBalances) {
-        console.log(`${entry.id}: ${entry.balanceOf.toString(2)}`);
+        console.log(`${entry.id}: ${entry.balanceOf.toLocaleString()}`);
```

Comment:
does `wei` have `toLocaleString`? I dont think so...

Reply:
Ah, yeah. Wanted the commas in the output, but will switch back to keep it simple.